app/components/widgets/forms/link-input.js

```diff
@@ -31,7 +31,6 @@ export default class LinkInput extends Component {
 
   @observes('protocol', 'address')
   protocolAddressObserver() {
-    if (!this.address) {return}
     const link = this.linkName?.toLowerCase();
```

Comment:
This was not letting us delete the whole `eventyay`, even in UI we removed `eventyay` but `e` was still not removed behind the scenes.

Reply:
Then please ensure this isn't happening - https://github.com/fossasia/open-event-frontend/issues/4947

Reply:
Looks good to me.

![Screenshot at 2020-09-10 19-13-08](https://user-images.githubusercontent.com/46647141/92738571-b0348800-f399-11ea-8c45-b17cdab752b3.png)